utils/build.ps1

```diff
@@ -1606,6 +1608,52 @@ function Build-LLVM([Platform]$Platform, $Arch) {
     }
 }
 
+function Build-Sanitizers([Platform]$Platform, $Arch) {
+  $BareTarget = $Arch.LLVMTarget.Replace("$AndroidAPILevel", "")
+  $LLVMDir = "$(Get-TargetProjectBinaryCache $Arch LLVM)\lib\cmake\llvm"
+  $InstallTo = "$($HostArch.ToolchainInstallRoot)\usr\lib\clang\19"
```

Comment:
Can we keep it like that? I can probably get it from running stage-1 Clang, or lit

Reply:
I think that we should extract this from the build, the resource dir changes on each rebranch, and this will complicate things.

Reply:
Ok, done. I think asking llvm-config would usually be the right way, but for cross-compiles we cannot execute it. We are asking `llvm-lit.py` now. Seems more reliable than parsing a CMake config file or a header somewhere in the build-tree.